.changeset/tricky-berries-retire.md

```diff
@@ -0,0 +1,9 @@
+---
+'@astrojs/starlight': minor
+---
+
+Updates `astro-expressive-code` dependency to the latest minor release (0.33).
+
+This unlocks support for [word wrap](https://expressive-code.com/key-features/word-wrap/) and [line numbers](https://expressive-code.com/plugins/line-numbers/), as well as updating the syntax highlighter to the latest Shiki release, which includes new and updated language grammars.
+
+See the [Expressive Code release notes](https://expressive-code.com/releases/) for more information on all contained changes.
```

Comment:
Maybe we explicitly call out “breaking changes” here as they are covered in the EC release notes?

```suggestion
See the [Expressive Code release notes](https://expressive-code.com/releases/) for more information including details of potentially breaking changes.
```

Reply:
Good idea, I committed your suggestion!